doc/export_model.md

```diff
@@ -0,0 +1,46 @@
+# Export Trained Models
+
+Once we train a model using SQLFlow via a statement like the following,
+
+```sql
+SELECT * FROM iris.train
+TO TRAIN DNNClassifier
+WITH model.n_classes=3,model.hidden_units=[10,20]
+INTO sqlflow_models.my_dnn_model;
+```
+
+the trained model `sqlflow_models.my_dnn_model` is saved in the model zoo.
```

Comment:
How about link model zoo to our model zoo doc?

Reply:
Changed model zoo -> database. The model is uploaded to the model zoo only if the user does so.